crates/typst/src/symbols/sym.rs

```diff
@@ -464,7 +464,13 @@ pub(crate) const SYM: &[(&str, Symbol)] = symbols! {
 
     // Miscellaneous.
     ballot: ['☐', x: '☒'],
-    checkmark: ['✓', light: '🗸'],
+    mark:[
+        '✓',
```

Comment:
This should presumably be `check: '✓',`

Reply:
Is it ok not to have a default?

Reply:
> Is it ok not to have a default?

There are plenty of examples of this in the file :)